### ISSUE TITLE
[RFE] Improve run-command's output formatting

### ISSUE THREAD

**Issue by blanquicet** [Azure/kubectl-aks#23]
## Current situation

All the `run-command` output is printed to the `stdout`, including the errors that the command executed in the node printed to the `stderr` in the node:

```bash
$ kubectl az run-command --node myNode "operation on files X and Y"
[stdout]
<Output of operation on file Y>
[stderr]
File X doesn't exits
```
## Ideal future situation

As discussed [here](https://github.com/Azure/kubectl-az/pull/14#discussion_r1151185555), it would be better to print to the actual `stdout` and `stderr`:

```bash
$ kubectl az run-command --node myNode "operation on files X and Y"
File X doesn't exits --> This should be printed to stderr
<Output of operation on file Y> --> This should be printed to stdout
```

**Comment by jainapurva** [comment]
Hi @blanquicet , I would like to take up this issue

**Comment by blanquicet** [comment]
> I would like to take up this issue
>
Hi @jainapurva, sure and welcome to the project!

**Comment by mqasimsarfraz** [comment]
@jainapurva How is going with issue? Just wanted to check in case you want any help from our side ? :)

**Comment by blanquicet** [comment]
@jainapurva friendly ping 🙂

**Comment by blanquicet** [comment]
Hi @jainapurva, it has been a while since we don't hear from you. Please notice that this issue is expected to be released in the next milestone: [v0.2.2](https://github.com/Azure/kubectl-aks/milestone/7) (due date: 17/07/23). So, please let us know if you're working on it. Otherwise, from tomorrow, we will have to start implementing it. Cheers! 🙂

**Comment by mqasimsarfraz** [comment]
I will jump on this issue for upcoming milestone. @jainapurva thanks again for volunteering and please feel free to pick any [open issue](https://github.com/Azure/kubectl-aks/issues) and reach us out in case you have questions. Always happy to help! 🙂
